### ISSUE TITLE
Manage items to be synced with tags

### ISSUE THREAD

**Issue by astromox** [kaiiiz/obsidian-raindrop-highlights-plugin#54]
This is a feature request. 

The ability to select what items are synced based on tags would be super useful. Currently, the ability to manage collections to be synced is great, but limited. 

For myself, I use both Raindrop and Obsidian for personal use and for work. I often use tags to organize items that are work related. 

Thanks!

**Comment by FeralFlora** [comment]
This would be great! 

As is, the options are either to limit synced bookmarks to highlighted items, or to download every bookmark in the collections you've chosen. It would be much more flexible if I could import items tagged with #review, #obsidian, #to-process, #to-do etc.. This would make the synced bookmarks more deliberate and actionable.

**Comment by kaiiiz** [comment]
I'm considering implementing this feature using the [search parameter in the `/rest/v1/raindrops` endpoint](https://developer.raindrop.io/v1/raindrops/multiple). This provides [a more generic way to build complex filters](https://help.raindrop.io/using-search/#operators).

**Comment by FeralFlora** [comment]
> I'm considering implementing this feature using the [search parameter in the `/rest/v1/raindrops` endpoint](https://developer.raindrop.io/v1/raindrops/multiple). This provides [a more generic way to build complex filters](https://help.raindrop.io/using-search/#operators).

This looks like a great approach. 

I think you could couple this with an option to import notes by searches rather than collections.
In this scenario, you could save several **named** searches. The bookmarks matching the search would be saved to directories that match the preconfigured name of the search.

To be clear, this was just an idea for an alternative behavior. I still think the search / filtering behavior should be applicable to standard imports into collection folders.

**Comment by FeralFlora** [comment]
@robertgr991 Yeah, this would be nice. However, it is necessary to consider the potential issue of what to do with bookmarks that fall under multiple searches? Making duplicate notes is not good, so what should be done in this case?
